platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -447,8 +450,12 @@ plugins {
 }
 ```
 
-The `buildCache.local.removeUnusedEntriesAfterDays` method is now deprecated.
-If set to a non-default value, this deprecated setting will take precedence over `Settings.caches.buildCache.setRemoveUnusedEntriesAfterDays()`.
+#### Groovy build script compilation build cache support is disabled
```

Comment:
@ljacomet should this be in "Other improvements" or "Build Cache improvements"?
Should these changes be made to the release branch or in the Google doc?

Reply:
This deprecations were before under `#### Allow version catalog plugin aliases without a version`, but they actually relate to change described under `#### Improved control of local build cache cleanup`. 

So I moved it to that section.